docs/examples/language_model/language_model.md

```diff
@@ -61,15 +61,17 @@ from mxnet.gluon.utils import download
 import gluonnlp as nlp
 ```
 
-#### Set environment
+#### Setup the environment (note that we should change num_gpus according to how many NVIDIA GPUs are available on the target machine)
 
 ```{.python .input}
 num_gpus = 1
 context = [mx.gpu(i) for i in range(num_gpus)] if num_gpus else [mx.cpu()]
 log_interval = 200
 ```
 
-#### Set hyperparameters
+#### Set the hyperparameters of the language model we are choosing
+
+Note that BPTT stands for "back propagation through time," and LR stands for learning rate. A link to more information on truncated BPTT can be found [here.](https://en.wikipedia.org/wiki/Backpropagation_through_time)
 
```

Comment:
move comma/period outside of quotes/brackets

Reply:
grammatically commas or punctuation for the main sentence should be within the quotes if at the end of a quote. https://www.grammarly.com/blog/quotation-marks/

Reply:
The examples on the link are quotations of conversations. However, "BPTT" is not part of conversations. See https://imgur.com/NSjHBAu